### DISCUSSION TITLE
What are "lighting peripherals"?

### DISCUSSION THREAD

**Discussion by ogami1972** [Drewol/unnamed-sdvx-clone#601] [Q&A] [answered]
Hello!

The page says "support for lighting peripherals" as something in progress. What are "lighting peripherals"?

**Comment by Drewol** [comment, marked as answer]
Some controllers have the ability to have their lights controlled by programs on the computer, [like this](https://www.youtube.com/watch?v=MCNMir-E_Rc).

The USC support is currently only on windows but it shouldn't be much work to get it running on linux.

**Comment by ogami1972** [comment]
Hello!

Cool, I have a Faucetwo, I will look into that. Not sure why you would want auto-play, other than like a kiosk-demo mode, but I like the idea of the side and top lights reacting to the beat, if that is actually what I'm seeing.

Thanks!

**Reply by Drewol** [reply]
yes, there's beat flashing and some button stuff in menus like the arcade, not just autoplay things.
